src/core/ConversationModel.cpp

```diff
@@ -277,7 +277,11 @@ QVariant ConversationModel::data(const QModelIndex &index, int role) const
     const MessageData &message = messages[index.row()];
 
     switch (role) {
-        case Qt::DisplayRole: return message.text;
+        case Qt::DisplayRole:
+          if (message.text.startsWith(QStringLiteral("/me "))) {
+            return QStringLiteral("* %1 %2").arg(m_contact->nickname(), message.text.mid(4, -1));
```

Comment:
Some thoughts:

If the message is just "/msg " then the resulting message is "* nick " which is a little odd.
 
Also these look like regular messages, it would be nice to do something like src/ui/LinkedText.cpp to alter the look of these messages so they stand out more (so return the  "/me " string here and morph it on the UI end instead of here - this allows some more flexibility later on if we want to add more actions.